docs/data-sources/endpoint_security_group.md

```diff
@@ -1,45 +1,94 @@
+  * `annotation` (annotation) - (string) The annotation of the Relation To Imported Contract object.
+  * `priority` (prio) - (string) The contract interface priority.
+  * `imported_contract_name` (tnVzCPIfName) - (string) The contract interface name.
+
+* `relation_to_intra_epg_contracts` - (list) A list of Relation To Intra Epg Contracts (ACI object [fvRsIntraEpg](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvRsIntraEpg/overview)) pointing to Contract (ACI Object [vzBrCP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/vzBrCP/overview)). This attribute is supported in ACI versions: 3.0(1k) and later.
+  * `annotation` (annotation) - (string) The annotation of the Relation To Intra Epg Contract object.
+  * `contract_name` (tnVzBrCPName) - (string) The binary contract name.
+
+* `relation_to_provided_contracts` - (list) A list of Relation To Provided Contracts (ACI object [fvRsProv](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvRsProv/overview)) pointing to Contract (ACI Object [vzBrCP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/vzBrCP/overview)). This attribute is supported in ACI versions: 1.0(1e) and later.
+  * `annotation` (annotation) - (string) The annotation of the Relation To Provided Contract object.
+  * `match_criteria` (matchT) - (string) The matched EPG type.
+  * `priority` (prio) - (string) The system class determines the quality of service and priority for the consumer traffic.
+  * `contract_name` (tnVzBrCPName) - (string) The provider contract name.
+
+* `relation_to_vrf` - (list) A list of Relation To VRF (ACI object [fvRsScope](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvRsScope/overview)) pointing to VRF (ACI Object [fvCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvCtx/overview)).
+  * `annotation` (annotation) - (string) The annotation of the Relation To VRF object.
+  * `vrf_name` (tnFvCtxName) - (string) Name.
 
-* `application_profile_dn` - (Required) Distinguished name of parent Application Profile object.
-* `name` - (Required) Name of the object Endpoint Security Group.
+* `relation_to_contract_masters` - (list) A list of Relation To Contract Masters (ACI object [fvRsSecInherited](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvRsSecInherited/overview)) pointing to Endpoint Security Group (ACI Object [fvESg](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvESg/overview)). This attribute is supported in ACI versions: 2.3(1e) and later.
+  * `annotation` (annotation) - (string) The annotation of the Relation To Contract Master object.
+  * `target_dn` (tDn) - (string) The distinguished name of the target.
```

Comment:
I am not sure if we should change this because it is a list of object fvRsSecInherited, which we translate into Relation To Contract Masters. So this is automatically deduced from the attribute name. 

For the attribute name we decided to prepend with relation_. Do you think we should change this to relations_ when there are more than 1 value allowed in the list? This is also a linguistic issue in the sense that we would need to determine the plural form of the word in code. Furthermore this would introduce breaking change sfor the already existing relation attribute names in our generated resources, and I am not sure that we should do this.

Reply:
You're right. It involves too much changes. There is no need to do that just for a small grammatical mistake that no one's gonna see. I just thought about overwriting the comment in the properties.yml file but I have not realized how many attributes were concerned by this.